closeio/contrib/django/views.py

```diff
@@ -34,6 +35,12 @@ def post(self, request, *args, **kwargs):
             logger.exception("CloseIO webhook request could not be dispatched.")
             return HttpResponseBadRequest()
 
+        # here we are checking which organization request comes from
+        organization_id = data.get('organization_id')
+        if organization_id != settings.CLOSEIO_ORGANIZATION_ID:
```

Comment:
This is kind of a breaking change. I would make this optional. If the setting is set, log the warning. What do you think?